### PR TITLE
 historical event page

### LINKED ISSUE #964: As a user I want to view historical events
In relation to issue #948, it would be great to create a gatsby page for historical events. The source code for the events page should be completely transferrable to this historical events page.

### PR DESCRIPTION
**Pull request by w8896699**
## Summary
A new page to show all past events

fixed #964

### CODE REVIEW

File: app-web/src/templates/events.js [hunk 1]
```diff
@@ -110,6 +110,10 @@ export const EventsPage = ({ data: { allEventbriteEvents, allDevhubTopic } }) =>
             <a href="https://github.com/bcgov/devhub-app-web/issues/new"> here</a> (requires a
             Github account).
           </p>
+          <p>
+            Click here if you interest in any
+            <a href="/past-events"> past events</a>.
```
Comment (patricksimonian) [inline app-web/src/templates/events.js]: Use `<Link>` component that leverages native gatsby utilities for routing

File: app-web/src/templates/events.js [hunk 2]
```diff
@@ -110,6 +110,10 @@ export const EventsPage = ({ data: { allEventbriteEvents, allDevhubTopic } }) =>
             <a href="https://github.com/bcgov/devhub-app-web/issues/new"> here</a> (requires a
             Github account).
           </p>
+          <p>
+            Click here if you interest in any
```
Comment (patricksimonian) [inline app-web/src/templates/events.js]: Let's talk about modifying this wording

File: app-web/src/templates/pastEvents.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  color: #1a5a96;
+  font-weight: 400;
+  text-transform: capitalize;
+  padding: 0 4px;
+  margin: 0 2px;
+  :hover {
+    cursor: pointer;
+  }
+`;
+export const formatEvents = events => {
```
Comment (patricksimonian) [inline app-web/src/templates/pastEvents.js]: looks like a duplicate function used in events

File: app-web/src/templates/events.js (hunk 1 above)
Comment (w8896699) [reply app-web/src/templates/events.js]: Can I use <ChevronLink> instead?

File: app-web/src/templates/pastEvents.js [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+                link={e.fields.standAlonePath}
+                event={e}
+              />
+            </Col>
+          ))}
+        </Row>
+      </CardContainer>
+      <LinkContainer>
+        {seeMoreResults && <SeeMoreP onClick={() => updateCount()}>See More Results</SeeMoreP>}
+        {/* {undefined && <ChevronLink to={undefined.to}>{undefined.text}</ChevronLink>} */}
```
Comment (patricksimonian) [inline app-web/src/templates/pastEvents.js]: what's this for?

**Pull request merged**